app/assets/stylesheets/shared.scss

```diff
@@ -289,3 +289,11 @@
 .approved {
   color: #339933
 }
+
+.event_small_col {
+  width: 36px;
+}
+
+.event_centered_col {
+  padding-left: 5px;
```

Comment:
Didn't use flexbox to center the text here because it somehow made an additional border in the cell that I couldn't get rid of, and I didn't want to waste time on it.